src/content/docs/docs/intro/web-monetization-extension.mdx

```diff
@@ -0,0 +1,225 @@
+:::
+
+### 2. Sign up with a compatible digital wallet
+
+Sign up with a Web Monetization-compatible [wallet provider](/docs/resources/op-wallets).
+
+After you've created your account, you'll be provided with a type of URL called a *wallet address*. You need this address in the next step. 
+
+```html title="Example wallet address"
+https://wallet.example.com/alice
+```
+
+### 3. Configure the extension
+
+<Steps>
+1. Open the extension and enter your wallet address in the given field.
+    <img src="/img/docs/extension/connection-setup.png" alt="Extension setup screen with wallet address and amount fields, renew monthly toggle, and connect button" style="max-width:300px" class="img-outline"/>
+2. Enter the amount to make available to spend. 
+3. Optionally select the **Renew monthly** toggle. If you choose to not renew monthly, you'll need to manually [add more funds](#adjust-your-budget) when your remaining balance is zero.
+4. Click **Connect**.
+5. Click **Agree** to allow the extension to connect to your wallet and add a key to your account.<br /><img src="/img/docs/extension/connection-wallet.png" alt="Consent screen to allow the extension to connect to your wallet provider with agree and decline buttons" style="max-width:300px" class="img-outline"/><br />If your wallet provider's website opens in your browser, proceed to the next step. If you receive a connection error, review the instructions for [resolving the error](#resolve-a-key-addition-failure).
+6. Sign in to your wallet account if you haven't already, then confirm the connection and budget amount. Below is an example confirmation screen.<br /><img src="/img/docs/extension/example-grant.png" alt="Example budget confirmation screen with accept and decline buttons" style="max-width:400px" class="img-outline" />A success screen appears when the connection is successful.<br /><img src="/img/docs/extension/connection-success.png" alt="Web page displaying a connection successful screen" style="max-width:400px" class="img-outline"/>
+</Steps> 
+
+## Adjust your budget 
+
+<Steps>
+1. Click the **Settings** icon in the upper-right of the extension.
+    <br /><img src="/img/docs/extension/settings-icon.png" alt="Extension showing a cog icon in the upper-right" style="max-width:300px" class="img-outline"/>
+2. Select the *Budget* tab.
+    <br /><img src="/img/docs/extension/budget-tab-full-budget.png" alt="Budget tab showing your remaining balance and options to change budget amount and renewal" style="max-width:300px" class="img-outline"/>
+3. Enter the new amount in the **Budget amount** field.
+4. Select the **Monthly** toggle to enable/disable monthly renewals of your budget.
+5. Click **Submit changes**. A new tab opens to your wallet provider. 
+6. Sign in to your wallet account if you haven't already, then confirm the new budget.
+</Steps>
+
```

Comment:
May I suggest moving this section after "make a one-time payment"?
It's closer to disconnect wallet (as adjust budget means disconnect and connect again with new budget), and it's not a regular operation to be this high in docs.